Makefile

```diff
@@ -27,7 +27,8 @@ stop: ## stop docker containers
 	- docker-compose down
 
 dc_build: ## rebuild docker compose containers
-	- docker-compose up --build -d
+	- make .docker_img_deps
+	- docker-compose up --build
```

Comment:
You stay use the job dedicated to development for prepare your deploy!  
Are you sure?

Reply:
the task `docker_img_deps` is for building the aws base image and js assets so the building process would be faster.
on dev the aws images is useless, the js assets will be overwritten by the node image.

Reply:
Can't you create a dedicated job to build prod to keep the dev build lighter?

Reply:
I could but I'd need to fix dev build as well, so not really, since dev is using the same images of prod

Reply:
ooooooooookkkkkkkkk